src/api/curriculumApi.ts

```diff
@@ -280,6 +287,30 @@ async function fetchCrossSubjectTopic(
   );
 }
 
+export async function fetchCrossSubjectTopicsByCode(
+  codes: string[],
+  language: string,
+  context: Context,
+): Promise<GQLReference[]> {
+  return Promise.all(
+    codes.map(async code => {
+      const response = await fetch(
+        `/grep/kl06/v201906/tverrfaglige-temaer-lk20/${code}`,
+        context,
+      );
+      const json: CrossSubjectTopic = await resolveJson(response);
+
+      const title = filterTextsForLanguage(json.tittel, language);
+
+      return {
+        code: json.kode,
+        id: json.kode,
+        title,
+      };
+    }),
+  );
+}
+
```

Comment:
Her blir det litt duplisert kode, så vet ikke om det er en enklere måte å få hentet ut riktig struktur på elementene:
1) Det er jo allerede en metode for å hente flere tverrfaglige temaer, men den trenger `codes: GQLElement[]` mens jeg bare har kodene som strings. Den baserer seg vel på strukturen til kompetansemål der de tverrfaglige temaene er strukturert som sånne elementer?
2) Skulle gjerne ha brukt `fetchGrepCodes()`, men så vidt jeg kan se har de metodene som bruker den en annen struktur på `tittel` i json-responsen, med `tittel: { tekst: Text[] }`. Responsen på de tverrfaglige kodene får bare `tittel: Text[]`. Men jeg ser på grep-apiet at de andre typen koder også har den siste strukturen, så skjer det kanskje noe mapping jeg har hoppet over som gjør at jeg kan bruke `fetchGrepCodes()` direkte?

Reply:
Tverrfaglige tema har et anna format i apiet enn dei andre typene så dette kan stå slik sånn eg ser det.